src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/prisonerprofile/PrisonerProfileService.kt

```diff
@@ -22,6 +22,7 @@ class PrisonerProfileService(
           bookingId = it.bookingId,
           startDate = it.bookingBeginDate.toLocalDate(),
           endDate = it.bookingEndDate?.toLocalDate(),
+          latestBooking = it.bookingSequence == 1,
```

Comment:
This is the main change - return a boolean indicating if this is the latest booking.